discovery-provider/src/queries/get_premium_track_signatures.py

```diff
@@ -289,11 +291,11 @@ def _unpack_metadata_account_for_metaplex_nft(data):
 
 
 def _get_metadata_account(mint_address: str):
-    return PublicKey.find_program_address(
+    return Pubkey.find_program_address(
         [
             b"metadata",
             bytes(METADATA_PROGRAM_ID_PK),
-            bytes(PublicKey(mint_address)),  # type: ignore
+            bytes(Pubkey(mint_address)),  # type: ignore
```

Comment:
Is this comment necessary?

Reply:
nope, thank you! i removed them all :D not sure why they were there before